internal/bminventory/inventory.go

```diff
@@ -3786,6 +3786,13 @@ func computeProxyHash(proxy *models.Proxy) (string, error) {
 	return fmt.Sprintf("%x", bs), nil
 }
 
+func validateClusterProxySettings(httpProxy, httpsProxy, noProxy, ocpVersion *string) error {
+	if noProxy != nil && *noProxy != "" && ocpVersion == nil {
+		return errors.Errorf("Cannot validate NoProxy: Unknown OpenShift version")
+	}
```

Comment:
Why do we need this?

Reply:
Answered in the other comment. Changing this.

Reply:
done